client/my-sites/themes/theme-options.js

```diff
@@ -328,6 +328,31 @@ function getAllThemeOptions( { translate, isFSEActive } ) {
 	};
 }
 
+export const getWooMyCustomThemeOptions = ( { translate, siteAdminUrl, siteSlug, options } ) => {
+	return {
+		assembler: {
+			key: 'assembler',
+			label: translate( 'Quick editing in the Assembler' ),
+			extendedLabel: translate( 'Quick editing in the Assembler' ),
+			getUrl: () => {
+				return `${ siteAdminUrl }admin.php?page=wc-admin&path=%2Fcustomize-store%2Fassembler-hub&customizing=true`;
+			},
+		},
+		customize: {
+			...options.customize,
+			label: translate( 'Advance customization in the Editor' ),
+			extendedLabel: translate( 'Advance customization in the Editor' ),
```

Comment:
👋 Should this be "Advanced" (as an adjective) instead of "Advance" (verb)?

Reply:
Thanks for flagging this! Confirming with the team. 🙏